<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiLoginDialog</class>
 <widget class="QWidget" name="UiLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="labelLoginName">
     <property name="text">
      <string>登录名</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="UiComboLine" name="lineLoginName"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelSecretId">
     <property name="text">
      <string>Sectet&amp;Id</string>
     </property>
     <property name="buddy">
      <cstring>lineSecretId</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineSecretId">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelSecretKey">
     <property name="text">
      <string>Sectet&amp;Key</string>
     </property>
     <property name="buddy">
      <cstring>lineSecretKey</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineSecretKey">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelRemark">
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineRemark">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkSaveSection">
     <property name="text">
      <string>记住会话</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="UiPushButton" name="btnLogin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UiComboLine</class>
   <extends>QLineEdit</extends>
   <header>src/fend/uicom/uicomboline.h</header>
  </customwidget>
  <customwidget>
   <class>UiPushButton</class>
   <extends>QPushButton</extends>
   <header>src/fend/uicom/uipushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
